builds/flu/flu_titers.py

```diff
@@ -70,5 +135,8 @@ def HI_export(process):
                       'avidity':process.HI_subs.compile_virus_effects(),
                       'substitution':process.HI_subs.compile_substitution_effects()}
         write_json(subs_model, prefix+'_titer_subs_model.json')
+        vaccine_distance_attribs.append('dTiterSub')
     else:
         print('Substitution model not yet trained')
+
+    vaccine_distance(process, vaccine_distance_attribs)
```

Comment:
I would remove the appends to the list of vaccine distance attributes above and make this call to the distance function from the process script. This will give maintainers more control over which distance attributes they calculate without having to change core titer functionality.